cloudapi/v1/cloudapi.proto

```diff
@@ -22,6 +26,8 @@ message Instance {
     V2RepoDetails v2_repo_details = 5;
   }
   ProjectDetails project_details = 6;
+  string status = 7;
+  repeated TaskDetails task_details = 8;
```

Comment:
it's a good chance to redesign our Instance model here. It still contains Cloud V1 fields which are no longer relevant.

Feel free to introduce breaking change, and that's fine!